tests/mockServer/mock.js

```diff
@@ -514,6 +518,80 @@ function deleteSecretKeys(req, res) {
 }
 
 function getProjectsSummaries(req, res) {
+  const currentDate = new Date()
+  const last24Hours = new Date(currentDate.getTime() - 24 * 60 * 60 * 1000)
+  const next24Hours = new Date(currentDate.getTime() + 24 * 60 * 60 * 1000)
+
+  // Pipelines
+  const possibleStatuses = ['Succeeded', 'Failed', 'Running']
+  const filteredPipelines24Hours = {}
+  for (const project in pipelines) {
+    const runs = pipelines[project].runs.filter(run => new Date(run.finished_at) > last24Hours)
+    if (runs.length > 0) {
+      const statusCounts = runs.reduce((counts, run) => {
+        counts[run.status] = (counts[run.status] || 0) + 1
+
+        return counts
+      }, {})
+
+      possibleStatuses.forEach(status => {
+        statusCounts[status] = statusCounts[status] || 0
+      })
+      filteredPipelines24Hours[project] = statusCounts
+    }
+  }
+
+  // Runs
+  const projectData = runs.runs
+    .filter(run => run.kind === 'run' && new Date(run.status.last_update).getTime() > last24Hours.getTime())
```

Comment:
you can remove .getTime() as for piplines, also here and for pipelines need to check if a time not bigger that 'now'